docs/root/configuration/http/http_filters/proto_message_extraction_filter.rst

```diff
@@ -1,23 +1,23 @@
 .. _config_http_filters_proto_message_extraction:
 
-Proto Message Extraction
+Proto message extraction
 ========================
 
 * gRPC :ref:`architecture overview <arch_overview_grpc>`
 * This filter should be configured with the type URL ``type.googleapis.com/envoy.extensions.filters.http.proto_message_extraction.v3.ProtoMessageExtractionConfig``.
 * :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.proto_message_extraction.v3.ProtoMessageExtractionConfig>`.
 
 ProtoMessageExtraction filter supports extracting gRPC
-requests/responses (proto messages) into google.protobuf.Struct and storing
-results in the dynamic metadata `envoy.filters.http.proto_message_extraction`
+requests/responses (proto messages) into ``google.protobuf.Struct`` and storing
+results in the dynamic metadata ``envoy.filters.http.proto_message_extraction``
 for later access.
 
 Use Case
 --------
 
 The ProtoMessageExtraction filter is particularly useful in scenarios where
 sensitive or detailed logging of gRPC requests and responses is required.
-In Client-Side Streaming or Server-Side Streaming, the filter can store the
+In client-side streaming or server-side Streaming, the filter can store the
```

Comment:
iiuc this is kinda borderline as a (gRPC) name but then when i grep the code base its always currently used lower case - so i think lets use that for consistency

Reply:
Sounds good!

Also,

`server-side Streaming` -> `server-side streaming`